### ISSUE TITLE
Unable to use cacheableResponse plugin

### ISSUE THREAD

**Issue by Nooshu** [GoogleChrome/workbox#2028]
**Library Affected**:
*workbox-sw 4.2.0 and 4.1.1 (those tested)*

**Browser & Platform**:
*Google Chrome v73  and v75 (Canary) for OSX*

**Issue or Feature Request Description**:
I'm trying to include the `workbox.cacheableResponse.Plugin` inside a `StaleWhileRevalidate` but it always throws an error:*

<img width="832" alt="Screen Shot 2019-04-11 at 23 03 43" src="https://user-images.githubusercontent.com/1223960/55996578-b5c30580-5caf-11e9-8dae-7287c350356f.png">

Code causing this can be seen below:
```js
workbox.routing.registerRoute(
  ({ event }) => event.request.destination === 'document',
  async ({ event }) => {
    try {
      return await new workbox.strategies.StaleWhileRevalidate({
        cacheName: savedPageCache,
        plugins: [
          new workbox.cacheableResponse.Plugin({
            statuses: [0, 200]
          }),
          new workbox.expiration.Plugin({
            // Only cache 20 requests.
            maxEntries: 40
          }),

        ]
      }).handle({ event });
    } catch (error) {
      console.error(error);
      return caches.match(offlinePage);
    }
  }
);
```
`workbox.expiration.Plugin` is being included and loads fine. Just seems to happen when using `cacheableResponse`.

Looks like it could be similar to [issue 1800](https://github.com/GoogleChrome/workbox/issues/1800).

**Comment by philipwalton** [comment]
I believe the issue you're facing is actually due to lazy imports. You can read more about the issue here: https://developers.google.com/web/tools/workbox/modules/workbox-sw#avoid_async_imports

**Comment by Nooshu** [comment]
Perfect, thanks @philipwalton yes that fixed it. For anyone with the same issue I included this directly after the initial script import:

```js
workbox.loadModule('workbox-cacheable-response');
```

What confused me was I was able to use the `workbox.expiration.Plugin` without an explicit import, but not the `workbox.cacheableResponse.Plugin`. My guess is that the expiration plugin is always loaded by default?
